packages/cloud_security_posture/manifest.yml

```
@@ -1,7 +1,7 @@
 format_version: 2.3.0
 name: cloud_security_posture
 title: "Security Posture Management"
-version: "1.5.0-preview23"
+version: "1.6.0-preview24"
```

if we want to include this change in 8.10
```suggestion
version: "1.5.0-preview24"
```